src/it/MCHECKSTYLE-357/pom.xml

```diff
@@ -55,9 +60,7 @@
 
                                     <!-- Annotations -->
                                     <module name="AnnotationUseStyle" />
-                                    <module name="MissingDeprecated">
```

Comment:
are we losing this part ?

Reply:
@eolivelli Please see https://checkstyle.org/releasenotes.html#Release_8.24
> MissingDeprecated: refactor to use javadoc AST 

and https://github.com/checkstyle/checkstyle/issues/4983 .

To me it looks the changes are just removing the property. property `skipNoJavadoc` was removed as the rewrite of the check will skip empty javadocs by default. If user wants to require javadocs we have a bunch of `MissingXxxJavadoc` checks.